woocommerce/woocommerce-android
# [WooExpress Removal] Remove store creation from the login flow

Part of: #11115 


⚠️ please don't merge yet, I'll merge everything when all PRs are approved.

### Description
This is the first part of WooExpress sign-up removal, it touches the login part, and has the following changes:
1. Update the prologue screen to remove the store creation option and replace it with a tertiary button to open a webpage.
2. Remove the "Create account" button from the wrong email screen.
3. Remove the store creation option from the site picker.

**This PR doesn't remove any logic code, the code cleanup will be handled separately.**

### Testing instructions
1. Check the login prologue, and confirm that it has a text button "Starting a new store?"
2. Click on the button, and confirm it opens the following [webpage](https://woo.com/hosting-solutions/)
3. Go back, and click on Login.
4. Enter the address of a Jetpack site.
5. In the email screen, enter a non valid email address.
6. In the "Email not found" screen, confirm that the "Create an account" button was removed.
7. Go back, and continue signing with a valid WordPress.com account.
8. Open the site picker.
9. Click on "Add a store"
10. Confirm it opens directly a screen for connecting an existing store. 

### Images/gif
<img width=360 src="https://github.com/woocommerce/woocommerce-android/assets/1657201/7f915b97-c41d-4511-96c2-3df1835d779a" />
